sale_block_no_stock/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.sale.block</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="sale.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='no_edit_order']" position="after">
+                <div class="col-12 col-lg-6 o_setting_box">
+					<div class="o_setting_right_pane">
+						<label for="sale_line_field_block" string="Blocking sales due to lack of stock" />
+                        <span
+                            class="fa fa-lg fa-building-o"
+                            title="Values set here are company-specific."
+                            groups="base.group_multi_company"
+                        />
+						<div class="text-muted">
+							Avoid selling more product than you have in stock or planned
+						</div>
+						<div class="content-group">
+							<div class="mt-2">
+                                <label
```

Comment:
It seems indentation went wild here!

Reply:
Why? Only want to write on the right_pane. And pre-commit do its things